src/site/content/en/blog/module-workers/index.md

```diff
@@ -0,0 +1,229 @@
+---
+title: "Threading the Web with Module Workers"
+subhead: |
+  Moving heavy lifting into background threads is now easier with JS Modules in Workers.
+date: 2019-12-17
+hero: hero.jpg
+alt: Module Workers
+authors:
+  - developit
+description: |
+  JavaScript is single-threaded, but Web Workers can be used to move expensive operations into
+  background threads. Module Workers bring modern JavaScript
+tags:
+  - post
+  - web-workers
+  - js-modules
+---
+
+<!--lint disable no-heading-punctuation heading-increment -->
+
+JavaScript is single-threaded, which means only one operation can be performed at a time. This is
```

Comment:
Please use active voice. Change 'only one operation can be performed at a time' to 'it can only perform one operation at a time.'